components/atom/tag/src/index.js

```diff
@@ -96,6 +101,10 @@ AtomTag.propTypes = {
    * To be used if href is defined
    */
   target: PropTypes.oneOf(['_self', '_blank', '_parent', '_top']),
+  /**
+   * To be used if href is defined
+   */
+  rel: PropTypes.oneOf(['nofollow', 'noopener', 'noreferrer']),
```

Comment:
W3C spec says that rel attribute value is a space-separated list of link types so it can have 1 or more of link types.

https://www.w3.org/TR/html401/struct/links.html#adef-rel

Reply:
Good point, I will do a refactor to change the prop to an array form